css/styled.css

```
@@ -435,8 +438,194 @@ body {
   padding-top: 25px;
   padding-left: 18px;
 }
+/* END information section */
 
 
+/* START featured section */
+
+.featured {
+  padding-top: 50px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+.featured__title {
+  color: #363535;
+  font-size: 24px;
+  font-weight: 700;
+  margin-bottom: 30px;
+}
+
+.featured__container {
+  display: flex;
+  gap: 20px;
+}
+
+.featured__products{
+  width: 60%;
+  background-color: var(--color-white);
+  box-shadow: 0px 0px 4px 0px rgba(54, 53, 53, 0.05);
+  border-radius: 12px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
+}
+
+.featured__product--line-bottom{
+  border-bottom: solid 1px #a6a6a660;
+  padding-bottom: 10px;
+}
+
+.featured__product--line-left {
+  border-right: solid 1px #a6a6a660;
+}
+
+.featured__product {
+  padding-top: 20px;
+  padding-left: 30px;
+  width: 230px;
+}
+
+.product--without-discount {
+  padding-top: 35px;
+}
+
+.featured__product-info {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  background-color: #05B999;
+  width: 100px;
+  border-radius: 4px 4px 4px 4px;
+  color: var(--color-white);
+  font-weight: 600;
+
+}
+
+.featured__product-info--red {
+  background-color: #EC4C4C;
+  width: 80px;
+}
+
+.featured__product-description {
+  color: #716C6C;
+  font-size: 16px;
+  margin-bottom: 20px;
+  padding-right: 10px;
+}
+
+.featured__product-price {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 20px;
+  font-weight: 700;
+}
+
+.featured__product-price--orange {
+  color: var(--brand-moving-orange);
+
+}
+
+.featured__product-price--gray {
+  font-size: 14px;
+  font-weight: 300;
+  color: #A6A6A6;
+  font-style: italic;
+  text-decoration-line: strikethrough;
+}
+
+.featured__product-detail {
+  border: 2px solid var(--brand-moving-orange);
+  border-radius: 12px;
+  width: 40%;
+  background-color: var(--color-white);
+  box-shadow: 0px 4px 12px 0px rgba(209, 77, 65, 0.10);
+  padding-top: 10px;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.featured__product-detail-info {
+  padding-top: 20px;
+}
+
+.featured__product-detail-container {
+  display: flex;
+}
+
+.product__detail-envio {
+  display: flex;
+  gap: 10px;
+  margin-top: 10px;
+  color: var(--layout-stone-grey);
+}
+
+.product__detail-envio--grey {
+  color: #D8D8D8
+}
+
+.product__detail-punctuation {
+  display: flex;
+  color: #716C6C;
+  gap: 2px;
+  font-weight: 400;
+}
+
+.product__detail-caption {
+  text-decoration: none;
+  color: #A6A6A6;
+  font-size: 12px;
+}
+
+.product__detail-name {
+  color: var(--layout-stone-grey);
+  font-size: 20px;
+  font-weight: 700;
+  margin-bottom: 60px;
+}
+
+.product__detail-price {
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.product__detail-price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 10px;
+  border-bottom: solid 1px #d8d8d805;
+}
```

Estas dos pueden ser una sola
```suggestion
.product__detail-price {
  font-size: 24px;
  font-weight: 700;
  display: flex;
  justify-content: space-between;
  padding-bottom: 10px;
  border-bottom: solid 1px #d8d8d805;
}
```